### PR TITLE
Db setup

### LINKED ISSUE #2: Create tests for Models
Test for Relations, Validations, Attributes

### PR DESCRIPTION
**Pull request by SK-Sam**
Necessary checkmarks:

    [X] All Tests are Passing
    
Type of change

    [X] New feature
       Notes: DB Setup, all models with relationships and validations set up
       
Testing Changes

    [x] No Tests have been changed
    [] Some Tests have been changed
    [] All of the Tests have been changed(Please describe what in the world happened)
    
Checklist:

    [] My code has no unused/commented out code
    [x] I have reviewed my code
    [x] I have commented my code, particularly in hard-to-understand areas
    [x] I have fully tested my code (Coverage 100%)
    [] Did you run Rubocop

Notes:            
Skipped Feature test until FE Users is complete

### CODE REVIEW

File: app/models/excursion.rb [hunk 1]
```diff
@@ -0,0 +1,13 @@
+class Excursion < ApplicationRecord
+  validates_presence_of :title
+  validates_presence_of :description
+  validates_presence_of :location
```
Comment (cunninghamge) [inline app/models/excursion.rb]: You can consolidate this into a single line if you're validating the same thing (ie presence) of multiple attributes

Comment (isabellevillasenor) [reply app/models/excursion.rb]: Good catch! We'll add this onto on current work

File: app/models/excursion.rb [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+
+  has_many :taggings
+  has_many :tags, through: :taggings
+  has_many :favorites
+  validates_presence_of :title
+  belongs_to :user
+
+  alias_attribute :author_id, :user_id
```
Comment (cunninghamge) [inline app/models/excursion.rb:9]: Is this actually being used anywhere?

Comment (SK-Sam) [reply app/models/excursion.rb:9]: You're right, removed line 9 to remove repeating line, thanks for checking that out @cunninghamge . Was line 10 or 12 questionable too?

File: app/models/excursion.rb (hunk 1 above)
Comment (SK-Sam) [reply app/models/excursion.rb]: Fixed and consolidated!

**Pull request merged**